WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostUiStateBuilder.kt

```diff
@@ -73,47 +73,6 @@ class ReaderPostUiStateBuilder @Inject constructor(
     private val readerPostTagsUiStateBuilder: ReaderPostTagsUiStateBuilder,
     @Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher
 ) {
-    @Suppress("LongParameterList")
-    suspend fun mapPostToUiState(
-        source: String,
-        post: ReaderPost,
-        isDiscover: Boolean = false,
-        photonWidth: Int,
-        photonHeight: Int,
-        postListType: ReaderPostListType,
-        onButtonClicked: (Long, Long, ReaderPostCardActionType) -> Unit,
-        onItemClicked: (Long, Long) -> Unit,
-        onItemRendered: (ReaderCardUiState) -> Unit,
-        onDiscoverSectionClicked: (Long, Long) -> Unit,
-        onMoreButtonClicked: (ReaderPostUiState) -> Unit,
-        onMoreDismissed: (ReaderPostUiState) -> Unit,
-        onVideoOverlayClicked: (Long, Long) -> Unit,
-        onPostHeaderViewClicked: () -> Unit,
-        onTagItemClicked: (String) -> Unit,
-        moreMenuItems: List<SecondaryAction>? = null
-    ): ReaderPostUiState {
-        return withContext(bgDispatcher) {
-            mapPostToUiStateBlocking(
-                source,
-                post,
-                isDiscover,
-                photonWidth,
-                photonHeight,
-                postListType,
-                onButtonClicked,
-                onItemClicked,
-                onItemRendered,
-                onDiscoverSectionClicked,
-                onMoreButtonClicked,
-                onMoreDismissed,
-                onVideoOverlayClicked,
-                onPostHeaderViewClicked,
-                onTagItemClicked,
-                moreMenuItems
-            )
-        }
-    }
-
     @Suppress("LongParameterList")
     fun mapPostToUiStateBlocking(
```

Comment:
It looks like this is still being used somewhere, is that correct? Doesn't it map to the old UI State for the old post UI?

Reply:
For some reason I thought it was being used, but it seems like it was just being used in the unit tests. Code updated.